src/EA.Weee.Web/Constant/BreadCrumbConstant.cs

```diff
@@ -3,6 +3,7 @@
     public static class BreadCrumbConstant
     {
         public static string AatfEvidence = "Manage AATF Evidence";
+        public static string AatfManageEvidence = "Manage AATF Evidence Notes";
         public static string AatfReturn = "Manage AATF returns";
```

Comment:
I think the Manage AATF Evidence activity one should just be "Manage evidence notes" if you could update

Reply:
wireframe says CCCI_00007 and content catalogue says "Manage evidence notes"